containers/server-image/Dockerfile

```diff
@@ -9,25 +9,24 @@ ARG PRODUCT_PATTERN="patterns-uyuni_server"
 
 # Add distro and product repos
 COPY add_repos.sh /usr/bin
-RUN sh add_repos.sh ${PRODUCT_REPO}
+RUN sh add_repos.sh
```

Comment:
Removing that optional parameter will add repos in the OBS build, which is not possible as OBS builds have no internet access